src/emails/templates/mjml.d.ts

```diff
@@ -186,13 +186,21 @@ declare namespace React.JSX {
         "background-position"?: CSSProperties["backgroundPosition"];
         "background-position-x"?: CSSProperties["backgroundPositionX"];
         "background-position-y"?: CSSProperties["backgroundPositionY"];
-        "background-repeat"?: CSSProperties["backgroundRepeat"];
+        "background-repeat"?: Extract<
```

Comment:
MJML threw a couple of errors, such as apparently not supporting `repeat-y` for `background-repeat`. Presumably because some email clients don't support it...